frontend/app/src/components/helper/PrioritizedListEntry.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <v-row align="center">
+    <v-col v-if="data.icon" cols="auto">
+      <adaptive-wrapper>
+        <v-img
+          :width="size"
+          contain
+          position="left"
+          :max-height="size"
+          :src="data.icon"
+        />
+      </adaptive-wrapper>
+    </v-col>
+    <v-col v-if="labels[data.identifier]" cols="auto">
+      {{ labels[data.identifier] }}
+    </v-col>
+    <v-col v-else cols="auto">
+      {{ toSentenceCase(data.identifier) }}
+    </v-col>
+  </v-row>
+</template>
+
+<script setup lang="ts">
+import { PropType } from 'vue';
+import AdaptiveWrapper from '@/components/display/AdaptiveWrapper.vue';
+import { AddressNamePriority } from '@/types/address-name-priorities';
+import { PriceOracle } from '@/types/price-oracle';
+import { PrioritizedListItemData } from '@/types/prioritized-list-data';
+import { PrioritizedListId } from '@/types/prioritized-list-id';
+import { toSentenceCase } from '@/utils/text';
+
+const props = defineProps({
+  data: {
+    required: true,
+    type: Object as PropType<PrioritizedListItemData<PrioritizedListId>>
+  }
+});
+
+const { data } = toRefs(props);
+
+const size = computed<string>(() => {
+  let defaultSize = '48px';
+  if (get(data).extraDisplaySize) {
+    return get(data).extraDisplaySize ?? defaultSize;
+  }
+  return defaultSize;
+});
+
+const { t } = useI18n();
+
+const labels: { [keys in PrioritizedListId]: string } = {
+  [PriceOracle.UNISWAP2]: t('oracles.uniswap_v2').toString(),
+  [PriceOracle.UNISWAP3]: t('oracles.uniswap_v3').toString(),
+  [PriceOracle.MANUALCURRENT]: t('oracles.manual_latest').toString(),
+  [AddressNamePriority.BLOCKCHAIN_ACCOUNT]: t(
+    'eth_address_book.hint.priority.list.blockchain_account_labels'
+  ).toString(),
+  [AddressNamePriority.ENS_NAMES]: t(
+    'eth_address_book.hint.priority.list.ens_names'
+  ).toString(),
+  [AddressNamePriority.ETHEREUM_TOKENS]: t(
+    'eth_address_book.hint.priority.list.ethereum_tokens'
+  ).toString(),
+  [AddressNamePriority.GLOBAL_ADDRESSBOOK]: t(
+    'eth_address_book.hint.priority.list.global_address_book'
+  ).toString(),
+  [AddressNamePriority.HARDCODED_MAPPINGS]: t(
+    'eth_address_book.hint.priority.list.hardcoded_mappings'
+  ).toString(),
+  [AddressNamePriority.PRIVATE_ADDRESSBOOK]: t(
+    'eth_address_book.hint.priority.list.private_address_book'
+  ).toString(),
+  blockchain: '',
+  coingecko: '',
+  cryptocompare: '',
+  fiat: '',
+  manual: '',
+  saddle: '',
+  defillama: ''
```

Comment:
We don't have labels for these ones hmm.
For some, it doesn't really make sense to have translatable resources.

Anyway, it looks way better than before thank you 😄

Reply:
These keys don't have labels as they are only used as they are but converted to sentence case. Because of the code `{ [keys in PrioritizedListId]: string }` I get an "property ... is missing in type" error when I leave them out.